### ISSUE TITLE
Unsupported major.minor version 52.0

### ISSUE THREAD

**Issue by fulopattila122** [wakatime/jetbrains-wakatime#47]
Hello,

I'm running PhpStorm 9.0 (#PS141-1912, 7 July), and today the wakatime plugin has updated itself to
4.0.14.

OS: Ubuntu 14.04
JRE: 1.7.0_79-b14 amd64
JVM: OpenJDK 64-Bit Server VM by Oracle Corporation
Installed Package versions: openjdk-7-jre: 7u79-2.5.5-0ubuntu0.14.04.2

Since the wakatime update the plugins fails with the following message:

```
Plugin 'com.wakatime.intellij.plugin' failed to initialize and will be disabled.  Please restart PhpStorm.

com.intellij.diagnostic.PluginException: com/wakatime/intellij/plugin/WakaTime : Unsupported major.minor version 52.0 [Plugin: com.wakatime.intellij.plugin]
    at com.intellij.ide.plugins.cl.PluginClassLoader.b(PluginClassLoader.java:130)
    at com.intellij.ide.plugins.cl.PluginClassLoader.a(PluginClassLoader.java:77)
    at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:66)
    at java.lang.ClassLoader.loadClass(ClassLoader.java:358)
    at java.lang.Class.forName0(Native Method)
    at java.lang.Class.forName(Class.java:274)
    at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentsRegistry.a(ComponentManagerImpl.java:427)
    at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentsRegistry.a(ComponentManagerImpl.java:417)
    at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentsRegistry.access$000(ComponentManagerImpl.java:403)
    at com.intellij.openapi.components.impl.ComponentManagerImpl.a(ComponentManagerImpl.java:108)
    at com.intellij.openapi.components.impl.ComponentManagerImpl.init(ComponentManagerImpl.java:87)
    at com.intellij.openapi.components.impl.stores.ApplicationStoreImpl.load(ApplicationStoreImpl.java:101)
    at com.intellij.openapi.application.impl.ApplicationImpl.load(ApplicationImpl.java:499)
    at com.intellij.openapi.application.impl.ApplicationImpl.load(ApplicationImpl.java:481)
    at com.intellij.idea.IdeaApplication.run(IdeaApplication.java:170)
    at com.intellij.idea.MainImpl$1$1$1.run(MainImpl.java:62)
    at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:312)
    at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:745)
    at java.awt.EventQueue.access$300(EventQueue.java:103)
    at java.awt.EventQueue$3.run(EventQueue.java:706)
    at java.awt.EventQueue$3.run(EventQueue.java:704)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:76)
    at java.awt.EventQueue.dispatchEvent(EventQueue.java:715)
    at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:362)
    at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:242)
    at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:161)
    at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:150)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:146)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:138)
    at java.awt.EventDispatchThread.run(EventDispatchThread.java:91)
Caused by: java.lang.UnsupportedClassVersionError: com/wakatime/intellij/plugin/WakaTime : Unsupported major.minor version 52.0
    at java.lang.ClassLoader.defineClass1(Native Method)
    at java.lang.ClassLoader.defineClass(ClassLoader.java:800)
    at java.lang.ClassLoader.defineClass(ClassLoader.java:643)
    at com.intellij.util.lang.UrlClassLoader._defineClass(UrlClassLoader.java:259)
    at com.intellij.util.lang.UrlClassLoader.defineClass(UrlClassLoader.java:255)
    at com.intellij.util.lang.UrlClassLoader._findClass(UrlClassLoader.java:231)
    at com.intellij.ide.plugins.cl.PluginClassLoader.b(PluginClassLoader.java:124)
    ... 30 more
```

I tried to uninstall/reinstall the plugin but no luck (so far).

PS.: I couldn't figure out how to install an older version of the plugin

**Comment by alanhamlett** [comment]
This looks like a regression of #25. I'll recompile the plugin with IntelliJ 13 and release a new version.

**Comment by alanhamlett** [comment]
Fixed with v4.0.15

**Comment by fulopattila122** [comment]
Woosh, that was fast :D
thanks a lot
